src/zoo/owl_zoo_log.ml

```diff
@@ -0,0 +1,85 @@
+(*
+ * OWL - an OCaml numerical library for scientific computing
+ * Copyright (c) 2016-2017 Liang Wang <liang.wang@cl.cam.ac.uk>
+ *)
+
+let dir = Sys.getenv "HOME" ^ "/.owl/zoo/"
```

Comment:
you have been using global dir everywhere in every module, spreading everywhere. If path is used so often, consider a separate module e.g. `owl_zoo_config` for accumulating this vars, and for making path such as extend_dir and etc.

Reply:
Thanks. I believe path should be named only once rather than using "~/.owl/zoo/" everywhere. A separate module is better solution than a global var.